src/plc.cpp

```diff
@@ -0,0 +1,6 @@
+#include "plc.h"
+
+PLC::PLC()
```

Comment:
If you don't need a custom constructor don't define one. Your class will become trivially constructable, copyable and movable.